flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/BuiltInFunctionTestBase.java

```diff
@@ -127,8 +128,19 @@ private static void testTableApiResult(
 
     private static void testTableApiError(Table inputTable, TableApiErrorTestItem testItem) {
         try {
-            inputTable.select(testItem.expression).execute();
-            fail("Error expected: " + testItem.errorMessage);
+            final TableResult tableResult = inputTable.select(testItem.expression).execute();
```

Comment:
can we move this refactoring to a separate hotfix commit? It has nothing to do with JSON_EXISTS and is useful without it.